pcsx2/CMakeLists.txt

```
@@ -1525,6 +1525,12 @@ if(COMMAND target_precompile_headers)
 	target_precompile_headers(PCSX2_FLAGS INTERFACE PrecompiledHeader.h)
 endif()
 
+if (UNIX OR WIN32)
```

```suggestion
if (NOT APPLE)
```
`UNIX` is true on macOS
I think the reason it's not actually renaming on macOS is because the bundle stuff doesn't look at OUTPUT_NAME, but I'd prefer if it was explicit

---

Good catch.